internal/builders/bazel/README.md

```diff
@@ -93,6 +93,9 @@ jobs:
 
 The `targets` are a set of space separated build targets to be built. Each target must include the `//` workspace root identifier and package target identifier (`:your_target`). Because of this each target should be of the form `//path/from/root/to/target:your_target`.
 
+Targets can also be referred to with general glob patterns such as `//src/...` or `//src/internal:all`. Note however, that support for artifacts that
+require runfiles is still currently in development and not available at this time.
```

Comment:
It might be nice to track this on an issue.

Reply:
I agree! Tracking it on this [issue](https://github.com/slsa-framework/slsa-github-generator/issues/2332)